bitnami/argo-cd/templates/notifications/bots/slack/deployment.yaml

```diff
@@ -0,0 +1,102 @@
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: notifications-slack-bot
+        {{- if .Values.notifications.bots.slack.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.notifications.bots.slack.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ include "argocd.notifications.bots.slack.serviceAccountName" . }}
+      {{- include "argocd.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.notifications.bots.slack.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.notifications.bots.slack.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.notifications.bots.slack.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.notifications.bots.slack.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.notifications.bots.slack.podAffinityPreset "component" "notifications" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.notifications.bots.slack.podAntiAffinityPreset "component" "notifications" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.notifications.bots.slack.nodeAffinityPreset.type "key" .Values.notifications.bots.slack.nodeAffinityPreset.key "values" .Values.notifications.bots.slack.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.notifications.bots.slack.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.notifications.bots.slack.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.notifications.bots.slack.priorityClassName }}
+      priorityClassName: {{ .Values.notifications.bots.slack.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.notifications.bots.slack.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.notifications.bots.slack.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: notifications-slack-bot
+          image: {{ include "argocd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.notifications.bots.slack.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.notifications.bots.slack.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.notifications.bots.slack.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.notifications.bots.slack.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+            - argocd-notifications
+            - bot
```

Comment:
Something that is not clear to me is where do we specify that the notifications are for slack?

Reply:
Also, how many bots could we deploy? One per notification service? If so, how can we configure that? My concern is that, if the changes are minimal between bots, we may be adding folder after folder with the different bots. Instead, it may make more sense to have a loop where we create the bots and services, that's why I'd like to know how this is configured.

Reply:
I followed the current configuration of the [community maintained ArgoCD chart](https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd/templates/argocd-notifications/bots/slack) and it's seems [it's the only one they currently support](https://argocd-notifications.readthedocs.io/en/latest/bots/overview/). If they add support for new bots, we can address that in the future, no?